### ISSUE TITLE
Possible design flaw in multiselect polls

### ISSUE THREAD

**Issue by ankush** [The-Commit-Company/Raven#863]
Currently raven splits a user's vote to equal amount for all options selected.

IMO in most cases the multi-select polls are done to find # of people voting for each individual option. 

E.g.

1. Which venue do you prefer for this event? 
2. Which weekday works for standup? 
3. What topping do you want on pizza for office party?
4. Which feature do you like the most from our last release?

In such cases, a single person voting 1 option will give a lot more weight to that single option.

**Comment by ankush** [comment]
Percentage should be based in total votes for each option IMO.

Weighted poll can be a different config, I can't think of many examples for that off the top of my head.

**Comment by nikkothari22** [comment]
> Percentage should be based in total votes for each option IMO.

Can you elaborate this?

As of now, percentage for an option is calculated as `total number of votes for said option / total number of votes across all options`.

**Comment by ankush** [comment]
% of each vote = total number of votes for that option / total # of users who voted.

**Comment by nikkothari22** [comment]
> % of each vote = total number of votes for that option / total # of users who voted.

Makes sense. The total would be higher than 100% overall - but that's fine I guess?
